### ISSUE TITLE
Z-Index Option in Advanced Tab

### ISSUE THREAD

**Issue by voodoocreativellc** [beaverbuilder/feature-requests#251]
I've been making a class of zindex to have better control of overlapping elements. This is something Elementor has built-in and it comes in handy often.

**Comment by gbissland** [comment]
There is a free 3rd party add-on which add z-index to the UI for modules/columns/rows.
https://sitespot.dev/downloads/beaver-builder-z-index/

**Comment by voodoocreativellc** [comment]
Wonderful thank you!

**Comment by Guenni1975** [comment]
Of course, it would still be great if it was integrated directly into the Beaver Builder so that an additional plugin doesn't have to be installed.
